### ISSUE TITLE
Evaluation: filter activated

### ISSUE THREAD

**Issue by LoickCh** [aharley/bev_baseline#32]
Hello,

Thank you for your great work. I have a question concerning your results, especially the visibility filter that seems to be activated in your training code (cf. _valid_bev_g_ during loss and metric calculation).

Did you use this filter for your tables in your paper ? It seems to have a big impact on metrics I got and some of the papers you compare with do not use this filter (cf. FIERY that overwrittes the filtering argument to False while training and evaluating).

Thank you in advance

**Comment by aharley** [comment]
Hi, thanks for checking. Yes, if a car is not visible from any camera viewpoint, we discard it. (As a single-frame method, I think this makes sense...) I think I see what you mean about FIERY: the default is visible-only, but they override this in some config files. That's a great catch. Our architecture takes a lot of pieces from FIERY, so you can maybe reproduce their results very closely, by using their recommended batch size and resolution, and efficientnet-b4.
